.azure/pipelines/ci.yml

```diff
@@ -19,6 +40,8 @@ jobs:
   parameters:
     # Ensures the alignment of branch name and deployment params
     buildArgs: '/warnaserror:BUILD1001'
+    ${{ if or(eq(variables['Build.Reason'], 'Schedule'), and(eq(variables['Build.Reason'], 'Manual'), eq(parameters.runCodeQL3000, 'true'))) }}:
+      runCodeQL3000: true
     afterBuild:
```

Comment:
Doubt we want this `task` when doing the `CodeQL3000` work. Suggest putting it under `${{ else }}:`

Reply:
This overlapping w/ the param name for the `task` was unintentional, I meant for it just to control some things in default-build. renamed.

Reply:
My point was about uploading the KoreBuild artifact. Don't do that in a `CodeQL3000` job.